extra/mdst/Makefile

```diff
@@ -25,8 +25,8 @@ ovc.mdst: sqlite2pb-ovc.py stations_pb2.py mdst.py ../../data/ovc_stations.db3
 shenzhen.mdst: csv2pb-szt.py stations_pb2.py mdst.py ../../data/szt/*.txt
 	python3 csv2pb-szt.py
 
-ezlink.mdst: csv2pb-ezlink.py stations_pb2.py mdst.py ../../data/ezlink/*.csv
```

Comment:
Is it intentional to delete the ezlink code?

Reply:
Yes. The real ezlink Makefile code is in data/ezlink now. This is just a mismerge on my part